src/main/java/org/jfrog/bamboo/util/deployment/LegacyDeploymentUtils.java

```diff
@@ -0,0 +1,42 @@
+package org.jfrog.bamboo.util.deployment;
+
+import com.atlassian.bamboo.deployments.execution.DeploymentTaskContext;
+import org.apache.commons.lang.StringUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jfrog.bamboo.deployment.ArtifactoryDeploymentConfiguration;
+
+/**
+ * Created by Dima Nevelev on 18/09/2018.
+ */
+public class LegacyDeploymentUtils {
+
+    public static String buildDeploymentSpec(@NotNull DeploymentTaskContext deploymentTaskContext) {
+        String repo = getRepoKey(deploymentTaskContext);
+        if (StringUtils.isBlank(repo)) {
+            // If repo is not configured, the task is not legacy
```

Comment:
I suggest rephrasing this log message to:
// If repo is not configured, no conversion is needed.

Reply:
Do you think we can rephrase the comment as suggested above?